src/screens/FeedScreen/Content.js

```diff
@@ -260,17 +264,26 @@ const Content = ({
   };
 
   return (
-    <Pressable
-      onLayout={handleHeightContainer}
-      onPress={isBlurredPost ? null : () => onPress()}
-      style={[styles.contentFeed, style]}>
+    <Pressable onPress={isBlurredPost ? null : () => onPress()} style={[styles.contentFeed, style]}>
       <BlurredLayer
         withToast={true}
         onPressContent={handleBlurredContent}
         isVisible={isBlurredPost}
         containerStyle={{
           flex: 1
         }}>
+        {handleContainerText().isShort && (
+          <LinearGradient
+            colors={['#184A57', '#275D8A']}
+            style={{
+              position: 'absolute',
+              top: 0,
+              left: 0,
+              right: 0,
+              bottom: 0
+            }}
+          />
+        )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A `LinearGradient` component has been added. Ensure that the colors and style provided to it are as expected. Also, make sure that the `handleContainerText().isShort` condition is correctly determining when to show the gradient.
```diff
+ <LinearGradient
+   colors={['#184A57', '#275D8A']}
+   style={{
+     position: 'absolute',
+     top: 0,
+     left: 0,
+     right: 0,
+     bottom: 0
+   }}
+ />
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->